daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/RuntimeData.scala

```diff
@@ -138,6 +138,14 @@ sealed abstract class TermRuntimeData(
   val maybeCheckBitOrderAndCharsetEv: Maybe[CheckBitOrderAndCharsetEv])
   extends RuntimeData {
 
+  /**
+   * Cyclic structures require initialization
+   */
+  lazy val initialize: Unit = {
+    partialNextElementResolver
```

Comment:
Should this be `partialNextElementResolver.force()`?

Reply:
So the arg, which is a delay is called partialNextElementResolverDelay.
The lazy val which is the value of this delay, forced is partialNextElementResolver.
So the initialize just asks for the value, that forces the delay. I'm trying to follow this style uniformly.